core/docs/d7yxc-operator_manual.md

```diff
@@ -102,6 +102,22 @@ To ensure that an SDK will be used with a compatible CLI and runner:
 1. If using a pre-installed CLI, install the CLI at that version as detailed in [Versioning](#versioning).
 1. Once the runner and/or CLI are setup, you are safe to upgrade your SDK to the newest version.
 
+### Can I run the Dagger Engine as a "rootless" container?
+
+No. "Rootless mode" means running the Dagger Engine as a container without the `--privileged` flag. In this case, the container would not have "root" privileges for all devices on the host system. Currently, the Dagger Engine cannot be run as a rootless container; network, platform and volume constraints related to rootless usage would significantly limit its capabilities and performance.
+
+#### Platform constraints
+
+The Dagger Engine uses Buildkit. In order to run it in rootless mode, it is necessary to use [Rootlesskit](https://github.com/rootless-containers/rootlesskit), an experimental tool that aims to replace the "root" user with a "fake root" user using user namespaces. However, depending on the host operating system, some features might not be fully supported.Since Dagger is intended to run consistently everywhere, introducing platform-specific incompatibilities would be contrary to this goal.
+
+#### Volume constraints
+
+The Dagger Engine relies on the `overlayfs` snapshotter for efficient operation of mounted filesystems. In order to run it in rootless mode, it is necessary to use [`fuse-overlayfs`](https://github.com/containers/fuse-overlayfs), a rootless implementation of `overlayfs`. However, optimal configuration of `fuse-overlayfs` depends on the kernel capabilities, its own configuration, `fuse` availability and upstream kernel patches specific to the host distribution. In the absence of all these, correctly configured, you will experience noticeable slowdown, either with `fuse-overlayfs` or the default snapshotter. Since Dagger aims to accelerate your CI/CD pipeline, reducing performance also reduces the value provided by Dagger and leads to unstable behaviour.
```

Comment:
```suggestion
The Dagger Engine relies on the `overlayfs` snapshotter for efficient construction of container filesystems. However, only relatively recent Linux kernel versions fully support `overlayfs` inside of rootless user namespaces. On older kernels, there are fallback options such as [`fuse-overlayfs`](https://github.com/containers/fuse-overlayfs), but they come with their own complications in terms of degraded performance and host-specific setup.

We've not yet invested in the significant work it would take to support+document running optimally on each kernel version, hence the limitation at this time.
```

Clarified a few details since technically `overlayfs` is possible to use on new enough kernels. Also trimmed some of the detail about fuse-overlayfs since I'm not sure it's relevant to readers at that level of detail.